recipe/meta.yaml

```diff
@@ -10,30 +10,43 @@ source:
   sha256: 3be9ca9d812e760969419e7288fef4d6700e7aa01d303aad4e56cd4ecb4f11ed
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
   # This packages uses libhdfs3, which is primarily used with HDFS on Linux.
-  skip: True  # [win]
+  # There is no libhdfs3 >=2.3 and its dependencies on s390x
+  skip: True  # [win or osx or s390x]
```

Comment:
Why skip on osx? I see we have built hdfs3 for osx-64 before?

Reply:
We haven't `libhdfs3` on osx-64 and it's pretty weird how hdfs3 was built without our libhdfs3

Reply:
`libhdfs3` https://anaconda.org/anaconda/libhdfs3/files
`hdfs3` https://anaconda.org/anaconda/hdfs3/files

Reply:
I removed osx from `skip`.

Reply:
> We haven't libhdfs3 on osx-64 and it's pretty weird how hdfs3 0.3.1 was built without our libhdfs3

That is totally weird, you are right about that. Apparently then we don't have a functional hdf3 on osx, so we may as well skip it. /me shrugs

Reply:
Rebuilt it with skip: `True  # [win or osx or s390x]`